File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/responses/PollResultsResponse.java
package com.eulersbridge.isegoria.network.api.responses;

import com.eulersbridge.isegoria.network.api.models.PollResult;
import com.squareup.moshi.Json;

import java.util.List;

public class PollResultsResponse {

    @Json(name = "nodeId")
    public long id;

    public long pollId;

    @Json(name = "answers")
    public List<PollResult> results;

}

File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/responses/PollResultsResponse.kt
package com.eulersbridge.isegoria.network.api.responses

import com.eulersbridge.isegoria.network.api.models.PollResult
import com.squareup.moshi.Json

data class PollResultsResponse (
    @field:Json(name = "nodeId")
    val id: Long,

    val pollId: Long,

    @field:Json(name = "answers")
    val results: List<PollResult>?
)